### DISCUSSION TITLE
How to download the DeDRM files?

### DISCUSSION THREAD

**Discussion by basx2016hub** [noDRM/DeDRM_tools#141] [Q&A]
Hi all,
I'm asking the n00b question because I don't know how I'd download the DeDRM files.

I'll assume installing it is similar to Alf Apprentice's original files? 

Any pointers would be much appreciated.

**Comment by ElleKayEm** [comment]
Go to the latest release page here:  https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

Download DeDRM_tools_10.0.3.zip.  Unzip it and you'll find the plugins and readme files.

**Reply by basx2016hub** [reply]
ElleKayEm,

Thanks!  I downloaded the latest version and now know where to look.

Thanks again!
